main.asm

```diff
@@ -294,6 +311,19 @@ Insert:
 		jsr WaitForNMI
 		jmp DumpBIOS									; then retry the file write
 
+InitFileHeader:
+		ldx #$00
+init_file_header_0:
+		lda FileHeaderR,x
+		sta FileHeader,x
+		inx
+		cpx #$11
+		bcc init_file_header_0
+		lda #$04
+		sta FileNum
+		rts
```

Comment:
```
InitFileHeader:
		ldx #$10
@loop:
		lda FileHeaderR,x
		sta FileHeader,x
		dex
		bpl @loop
		lda #$04
		sta FileNum
		rts
```
It's better to structure the loop direction so that the status flags after a decrement/increment determines when to terminate the loop. Since we want indices $00~$10, starting at $10 and decrementing until it underflows to $FF (setting the negative flag) is better here.